### PR TITLE
Handle IMA library as an imported target

### PR DESCRIPTION
**Pull request by pmatilai**
Most of our dependencies are imported targets already, so while for just one site this isn't actually any shorter in terms of LoC it's more consistent with the surroundings in the consuming site, and more so if we ever need to link to this from some other place.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -194,11 +194,14 @@ endif()
 
 if (WITH_IMAEVM)
 	list(APPEND REQFUNCS lsetxattr)
-	find_path(IMA_INCLUDE_DIR NAMES imaevm.h)
-	find_library(IMA_LIBRARY NAMES imaevm)
-	if (NOT (IMA_INCLUDE_DIR AND IMA_LIBRARY))
-		message(FATAL_ERROR "imaevm enabled but library not present")
-	endif()
+	find_path(IMA_INCLUDE_DIR NAMES imaevm.h REQUIRED)
+	find_library(IMA_LIBRARY NAMES imaevm REQUIRED)
```
Comment (dmnks) [inline CMakeLists.txt:198]: REQUIRED was only added in cmake 3.18, though. But without it, one has to use these awkward checks and custom error messages...

Comment (dmnks) [reply CMakeLists.txt:198]: Which makes me think if we shouldn't just bump our minimal cmake version?

Comment (dmnks) [reply CMakeLists.txt:198]: ... aaaand done. 3.18 it is :smile:

**Comment by pmatilai** [comment]
The IMA target was actually broken, reverted in 589fb0e3aa15683cc71931da95b328e57a9c58b6

Thanks @dmnks for spotting.

**Pull request merged**